### DISCUSSION TITLE
Support for Amazon Bedrock Prompt Caching

### DISCUSSION THREAD

**Discussion by suin** [cline/cline#858] [Feature Requests]
With Amazon Bedrock's announcement today of their Prompt Caching feature, it would be nice to add support for it in Cline. Since Cline already handle caching for providers like Anthropic, extending this to Bedrock would provide a consistent experience across platforms. 

Reference: [Amazon Bedrock Prompt Caching Documentation](https://docs.aws.amazon.com/bedrock/latest/userguide/prompt-caching.html)

**Comment by sreimchen** [comment]
Your feature request is a duplicate: https://github.com/cline/cline/discussions/872

**Comment by elemental-mind** [comment]
Please close as duplicate.
